infra/templates/az-isolated-service-single-region/tests/integration/integration_test.go

```diff
@@ -0,0 +1,91 @@
+package test
+
+import (
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	httpClient "github.com/gruntwork-io/terratest/modules/http-helper"
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	"github.com/microsoft/cobalt/test-harness/infratests"
+)
+
+var region = "eastus2"
+
+// see note about static workspace in test case below
+var workspace = "cobalt-isolated-testing"
+
+var admin_subscription = os.Getenv("TF_VAR_admin_subscription_id")
+var ase_name = os.Getenv("TF_VAR_app_service_environment_name")
+var ase_resource_group = os.Getenv("TF_VAR_app_service_environment_resource_group")
+
+var tfOptions = &terraform.Options{
+	TerraformDir: "../../",
+	Upgrade:      true,
+	Vars: map[string]interface{}{
+		"resource_group_location": region,
+		"ase_subscription_id":     admin_subscription,
+		"ase_name":                ase_name,
+		"ase_resource_group":      ase_resource_group,
+		"app_service_name": map[string]interface{}{
+			"cobalt-backend-api-1": "appsvcsample/static-site:latest",
+			"cobalt-backend-api-2": "appsvcsample/static-site:latest",
+		},
+	},
+	BackendConfig: map[string]interface{}{
+		"storage_account_name": os.Getenv("TF_VAR_remote_state_account"),
```

Comment:
Suggestion: may want to look at envy for working with envvars. Adds a lot of functionality and error handling.

https://github.com/gobuffalo/envy

Reply:
Thanks for sharing this. We discussed that this is a good path moving forward but no need to switch at the moment.